test/htmltojsx-test.js

```diff
@@ -136,6 +136,42 @@ describe('htmltojsx', function() {
       expect(converter.convert('<div style="font-size: 12pt">Test</div>').trim())
         .toBe('<div style={{fontSize: \'12pt\'}}>Test</div>');
     });
+	
+	it('should convert vendor-prefix "style" attributes', function() {
```

Comment:
Fix the spacing here please - Looks like the `it` lines use tabs but everything else uses spaces